var/spack/repos/builtin/packages/libint/package.py

```diff
@@ -45,6 +58,11 @@ def autoreconf(self, spec, prefix):
         aclocal('-I', 'lib/autoconf')
         autoconf()
 
+        if '@2.6.0:' in spec:
+            # skip tarball creation and removal of dir with generated code
+            sed = which('sed')
+            sed('-i -e', 's|tgz$||', 'export/Makefile')
```

Comment:
You can use `filter_file` if you don't want to rely on the presence of `sed`

Reply:
done, thanks!